arangod/RocksDBEngine/RocksDBMetadata.cpp

```diff
@@ -335,9 +343,34 @@ Result RocksDBMetadata::serializeMeta(rocksdb::WriteBatch& batch,
   }
 
   const rocksdb::SequenceNumber maxCommitSeq = committableSeq(appliedSeq);
+
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
+  // simulate another transaction coming along and trying to commit while
+  // we are serializing
+  TransactionId trxId = TransactionId::none();
+  
+  TRI_IF_FAILURE("TransactionChaos::randomSleep") {
```

Comment:
Should we use a different failure point here? This is no "randomSleep".

Reply:
Done